prowler/lib/cli/parser.py

```diff
@@ -349,25 +345,7 @@ def __init_aws_parser__(self):
             "--allowlist-file",
             nargs="?",
             default=None,
-            help="Path for allowlist yaml file. See example prowler/config/allowlist.yaml for reference and format. It also accepts AWS DynamoDB Table or Lambda ARNs or S3 URIs, see more in https://docs.prowler.cloud/en/latest/tutorials/allowlist/",
-        )
-        # Based Scans
-        aws_based_scans_subparser = aws_parser.add_argument_group("AWS Based Scans")
-        aws_based_scans_parser = (
-            aws_based_scans_subparser.add_mutually_exclusive_group()
-        )
-        aws_based_scans_parser.add_argument(
-            "--resource-tags",
-            nargs="+",
-            default=None,
-            help="Scan only resources with specific AWS Tags (Key=Value), e.g., Environment=dev Project=prowler",
-        )
-        aws_based_scans_parser.add_argument(
-            "--resource-arn",
-            nargs="+",
-            type=arn_type,
-            default=None,
-            help="Scan only resources with specific AWS Resource ARNs, e.g., arn:aws:iam::012345678910:user/test arn:aws:ec2:us-east-1:123456789012:vpc/vpc-12345678",
```

Comment:
Same here. Review this carefully, we cannot remove this, also is not related to this PR.

Reply:
Done!